packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -78,8 +78,8 @@
 :host(:not([uswds='true'], [uswds=''])[secondary]:not([secondary='false'])) button:active,
 :host(:not([uswds='true'], [uswds=''])[secondary]:not([secondary='false'])) button:hover {
   background-color: var(--color-gray-cool-light);
-  box-shadow: inset 0 0 0 2px var(--color-primary-darker);
-  color: var(--color-primary-darker);
+  box-shadow: inset 0 0 0 2px var(--color-primary-dark);
+  color: var(--color-primary-dark);
 }
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)